docs/_future/create_new_app.md

```diff
@@ -0,0 +1,765 @@
+</div>
+
+![Request Time Off app](../assets/take-your-time-demo.gif "Request Time Off app")
+
+When you’re finished, you’ll have this ⚡️[Bolt for JavaScript Request Time Off app](https://github.com/slack-samples/bolt-js-request-time-off) to run, modify, and make your own.
+
+If you already have an existing Bolt for JavaScript application that you'd like to migrate to the next-generation platform, check out the [Migration guide](/bolt-js/future/migrate-existing-app) instead.
+
+---
+### Prerequisites {#prerequisites}
+
+In order to create your app, make sure you've first followed all steps in the [Getting Started guide](/bolt-js/future/getting-started) to set up and install the dependencies.
+
+---
+
+### Create an app {#create-an-app}
+
+Before you start developing with Bolt, you'll want to create a Slack app. 
+
+To create the app, you'll run the following command:
+```
+slack create my-app -t slack-samples/bolt-js-request-time-off
+```
+This command creates an app through the CLI by cloning a specified template. In this case, the template is the [Bolt for JavaScript Request Time Off](https://github.com/slack-samples/bolt-js-request-time-off) application. 
+
+The application will take a few moments to set up and clone the repository. Once the app is successfully created, you should see a message like this:
+```
+✨ my-app successfully created
+
+🧭 Explore your project's README.md for documentation and code samples, and at any time run hermes help to display a list of available commands
+
+🧑‍🚀 Follow the steps below to try out your new project
+
+1️⃣  Change into your project directory with: cd my-app
+
+2️⃣  Develop locally and see changes in real-time with: slack run
+
+3️⃣  When you're ready to deploy for production use: slack deploy
+
+🔔 If you leave the workspace, you won’t be able to manage any apps you’ve deployed to it. Apps you deploy will belong to the workspace even if you leave the workspace
+```
+
+You can now explore the `my-app` directory!
+
+---
+### Set up your trigger {#setup-trigger}
+
+In order to utilize the pre-existing functionality in the Request Time Off app, you'll need to create a trigger prior to running your app.
+
+First, make sure you're in the project directory in your command line: `cd my-app`
+
+Then, run the following command to create a trigger using the `triggers/link-shortcut.json` configuration file. This file was brought into your project from the Bolt for JavaScript Request Time Off template and is used to initialize your trigger:
+```
+slack triggers create --trigger-def "triggers/link-shortcut.json"      
+```
+
+The above command will create a trigger for the selected workspace. Once the trigger is successfully created, you should see an output like this:
+```
+⚡ Trigger created
+   Trigger ID:   [ID]
+   Trigger Type: shortcut
+   Trigger Name: Take Your Time
+   Shortcut URL: https://slack.com/shortcuts/[ID]/[Some ID]
```

Comment:
```suggestion
   URL: https://slack.com/shortcuts/[ID]/[Some ID]
```
Output is now just URL I believe. Inevitably we'll get drift on these, but that's the nature of docs I suppose